Digipost.Api.Client.DataTypes.Core/Resources/XSD/datatypes.xsd

```diff
@@ -1,474 +1,1337 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://api.digipost.no/schema/datatypes"
-           elementFormDefault="qualified" targetNamespace="http://api.digipost.no/schema/datatypes" version="1.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://api.digipost.no/schema/datatypes" elementFormDefault="qualified" targetNamespace="http://api.digipost.no/schema/datatypes" version="1.0">
+      
+  
```

Comment:
Vanskelig å lese diff når git mener hele filen er endret. Også fordi det er lagt til to tomme linjer mellom hvert element. Noen grunn til dette? Har generatoren endret oppførsel siden sist den ble kjørt?

Reply:
Jeg dupliserte både XSD-fila og XML-eksemplet direkte fra digipost-data-types. Tenkte at også XML-fila bør være identisk, siden den også er generert fra Java-koden i digipost-data-types. Hvis det ikke er tilfellet, kan jeg patche den opp for hånd i dette repoet.

Problemet med tomme linjer har oppstått i digipost-data-types. Vet ikke hva det skyldes. Det er jo bottom-up kodegenerering fra Java, kan være en eller annen innstilling som har kommet på avveie, eller fått en uheldig default i mitt lokale oppsett (generert kode committes fra lokalt bygg).